src/modules/category/selectors.ts

```diff
@@ -1,24 +1,74 @@
 import { createSelector } from 'reselect'
 import { RootState } from 'modules/common/types'
 import { CategoryState } from 'modules/category/reducer'
-import { AssetState } from 'modules/asset/reducer'
 import { FullCategory } from 'modules/category/types'
+import { getData as getAssetPacks } from 'modules/assetPack/selectors'
 import { getData as getAssets } from 'modules/asset/selectors'
+import { AssetState } from 'modules/asset/reducer'
+import { AssetPackState } from 'modules/assetPack/reducer'
+import { Asset } from 'modules/asset/types'
 
 export const getState: (state: RootState) => CategoryState = state => state.category
 
-export const getData: (state: RootState) => CategoryState['data'] = state => getState(state).data
+export const getSelectedAssetPackId = (state: RootState) => getState(state).selectedAssetPackId
 
-export const isLoading: (state: RootState) => boolean = state => getState(state).loading.length > 0
+export const getSearch = (state: RootState) => getState(state).search
 
-export const getError: (state: RootState) => CategoryState['error'] = state => getState(state).error
+const isSearchResult = (asset: Asset, search: string) => {
```

Comment:
I don't get why is this here, it's not checking _only_ the category, so wouldn't this be a responsibility of `assetPack`? or maybe `ui/search` ?

Reply:
https://github.com/decentraland/builder/pull/106#discussion_r253470051